src/native/managed/cdac/Microsoft.Diagnostics.DataContractReader.Contracts/Contracts/StackWalk/Context/RISCV64Context.cs

```diff
@@ -0,0 +1,218 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Runtime.InteropServices;
+using Microsoft.Diagnostics.DataContractReader.Contracts.StackWalkHelpers.RISCV64;
+
+namespace Microsoft.Diagnostics.DataContractReader.Contracts.StackWalkHelpers;
+
+/// <summary>
+/// RISC-V 64-bit specific thread context.
+/// </summary>
+[StructLayout(LayoutKind.Explicit, Pack = 1)]
+internal struct RISCV64Context : IPlatformContext
+{
+    [Flags]
+    public enum ContextFlagsValues : uint
+    {
+        CONTEXT_RISCV64 = 0x00800000,
```

Comment:
Looking at [dbgtargetcontext.h](https://github.com/dotnet/runtime/blob/f8b5e9cde2e1046a3cf7d1a32035ac1dee5ad4a2/src/coreclr/debug/inc/dbgtargetcontext.h#L553), the RISCV64 platform context bit looks like it should be `0x01000000`
```suggestion
        CONTEXT_RISCV64 = 0x01000000,
```